CODE/style.css

```diff
@@ -0,0 +1,130 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #d2d3d3;
+
+}
+
+/* pull QUote*/
+blockquote {
+    text-align: left;
+    margin: 0;
+}
+
+
+header {
+    background-image: url(images/maxresdefault.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+/*Title and Author*/
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h2 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 45px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h3 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 30px;
+    text-align: left;
+}
+
+/*article text*/
+p1 {
+    columns: 100 px 1;
```

Comment:
This is currently broken and not working for a few reasons:
1. the space between 100 and px (ie, you want `100px`)
2. the column count (the last item in this value) is 1, which is the same as not having columns at all.
3. Instead of declaring `columns` on each individual paragraph, you instead want to wrap **all** the `p` elements in the story in a `div` like so:
```html
<div class="column-wrapper">
  <p>Content here.</p>
  <p>Content here.</p>
  <p>Content here.</p>
  <p>Content here.</p>
</div>
```

And then attach this CSS to the newly created class:
```css
.columns {
    column-count: 2;
    column-gap: 55px;
}
```
This would give you something more like this:
![rafael-columns](https://user-images.githubusercontent.com/2653980/53538333-3dc3c600-3adb-11e9-8a08-313775718679.jpg)

Learn more about CSS columns [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns/Using_multi-column_layouts).